ConsoleApp1/Program.cs

```diff
@@ -1,15 +1,118 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
-namespace ConsoleApp1
+namespace Lesson1
 {
     class Program
     {
         static void Main(string[] args)
         {
+            //Exercise 1
+            Console.WriteLine("Exercise 1. Press any key");
+            Console.ReadKey();
+            Console.WriteLine("Enter x");
+            int x = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter y");
+            int y = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter z");
+            int z = Convert.ToInt32(Console.ReadLine());
+
+            if (y < 0)
+            {
+                Console.WriteLine("!!!y < 0!!! the square root of a negative number does not exist.");
+            }
+
+            else if(z == 0)
+            {
+                Console.WriteLine("!!!z = 0!!! cannot be divided by 0.");
+            }
+
+            else
+            {
+                var fexpression = (int)((x + y) * (Math.Pow(z, 2) + 1));
+                Console.WriteLine("First expression: " + fexpression);
+
+                var sexpression = ((x % z) - 1) * (Math.Sqrt(y));
+                Console.WriteLine("Second expression: " + (Math.Round(sexpression, 2)));
+
+                var texpression = ((x + y) * (y + z)) / (Math.Pow(z, 3));
+                Console.WriteLine("Third expression: " + (Math.Round(texpression, 2)));
+                Console.ReadKey();
+            }
+
+            //Exercise 2
+            Console.WriteLine("Exercise 2. Press any key");
+            Console.ReadKey();
+            Console.WriteLine("Enter length");
+            int length = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter width");
+            int width = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter height");
+            int height = Convert.ToInt32(Console.ReadLine());
+
+            if(length < 0 ^ width < 0 ^ height < 0)
```

Comment:
Why we need bit XOR there?
if you want to check all three sides there you need to use &&

Reply:
Sorry, I mixed up the operators...